style.css

```diff
@@ -7,23 +7,25 @@
 
 body {
 	height: 100vh;
```

Comment:
Why is body height set to 100vh when we don't have enough content on page to get to scrolling state?